core/cloudflow-akka/src/main/scala/cloudflow/akkastream/AkkaStreamletContext.scala

```diff
@@ -84,7 +84,7 @@ trait AkkaStreamletContext extends StreamletContext {
   private[akkastream] def metricTags(): Map[String, String]
 }
 
-final class StreamletContextException() extends Exception("StreamletContext can only be accessed from the `createLogic()` method.")
+final class AkkaStreamletContextException() extends StreamletContextException
```

Comment:
```suggestion
final class AkkaStreamletContextException() extends StreamletContextException("The SparkStreamletContext can only be accessed from within the streamlet logic.")
```

Reply:
@RayRoestenburg you seem to be suggesting a spark error message on the Akka exception...

Reply:
Thanks, fixed.